ingestion/src/metadata/ingestion/source/dashboard/redash/metadata.py

```diff
@@ -57,12 +81,64 @@ def create(cls, config_dict: dict, metadata_config: OpenMetadataConnection):
             )
         return cls(config, metadata_config)
 
+    def prepare(self):
+        """
+        Fetch the paginated list of dashboards and tags
+        """
+
+        self.dashboard_list = self.client.paginate(self.client.dashboards)
+
+        # Collecting all the tags
+        for dashboard in self.dashboard_list:
+            self.tags.extend(dashboard.get("tags") or [])
+
+    def yield_tag(self, *_, **__) -> OMetaTagAndClassification:
+        """
+        Fetch Dashboard Tags
+        """
+        for tag in self.tags:
+            try:
+                classification = OMetaTagAndClassification(
+                    classification_request=CreateClassificationRequest(
+                        name=REDASH_TAG_CATEGORY,
+                        description="Tags associates with redash entities",
+                    ),
+                    tag_request=CreateTagRequest(
+                        classification=REDASH_TAG_CATEGORY,
+                        name=tag,
+                        description="Redash Tag",
+                    ),
+                )
+                yield classification
+                logger.info(f"Classification {REDASH_TAG_CATEGORY}, Tag {tag} Ingested")
+            except Exception as exc:
+                logger.debug(traceback.format_exc())
+                logger.warning(f"Error ingesting tag {tag}: {exc}")
+
+    def get_tag_labels(self, tags: Optional[List[str]]) -> Optional[List[TagLabel]]:
+        if tags:
+            return [
+                TagLabel(
+                    tagFQN=fqn.build(
+                        self.metadata,
+                        Tag,
+                        classification_name=REDASH_TAG_CATEGORY,
+                        tag_name=tag,
+                    ),
+                    labelType="Automated",
```

Comment:
@OnkarVO7 can we import these value from `tagLabel.py` or have a constant define above